rpcs3/rpcs3qt/main_window.cpp

```diff
@@ -1330,6 +1330,10 @@ void main_window::CreateConnects()
 		m_gameListFrame->SetShowHidden(checked);
 		m_gameListFrame->Refresh();
 	});
+	connect(ui->showCompatibilityInGridAct, &QAction::triggered, [=](bool checked)
+	{
+		m_gameListFrame->SetShowCompatibilityInGrid(checked);
```

Comment:
if you put SetShowCompatibilityInGrid into the Q_SLOT section in game_list_frame.h then you can simply use it in one line like this:

`connect(ui->showCompatibilityInGridAct, &QAction::triggered, m_gameListFrame, &game_list_frame::SetShowCompatibilityInGrid);`